howso/hyperparameters.amlg

```diff
@@ -224,60 +224,14 @@
 			)
 		)
 
-		(declare (assoc deviations (get baseline_hyperparameter_map "!analyzeFeatureDeviations") ))
+		(if (= (first (get baseline_hyperparameter_map "paramPath")) ".default")
```

Comment:
you can simplify this imo, since you're already doing a (get) opcode, may as well drop the (first) opcode and just provide the path ["paramPath" 0] instead

Reply:
yep, good eye